src/oxoPatterns/oxoPattern1.java

```diff
@@ -1,31 +1,30 @@
-package oxoPatterns;
+package src.oxoPatterns;
 
 import java.util.Scanner;
 
 public class oxoPattern1 {
     public static void main(String[] args) {
-        Scanner scan = new Scanner(System.in);
+        Scanner userInputScanner = new Scanner(System.in);
         try{
             System.out.print("Line:");
-            int n = scan.nextInt();
+            int userInputNumbersOfLines = userInputScanner.nextInt();
             System.out.print("Column:");
-            int c = scan.nextInt();
-            for(int i = 1; i <= n; i++){
+            int userInputNumbersOfColumns = userInputScanner.nextInt();
+            for(int i = 1; i <= userInputNumbersOfLines; i++){
                 if(i % 2 == 0){
-                    for(int j = 1; j <= c; j++){
+                    for(int j = 1; j <= userInputNumbersOfColumns; j++){
                         System.out.print("xo");
                     }
                 } else {
-                    for(int j = 1; j <= c; j++){
+                    for(int j = 1; j <= userInputNumbersOfColumns; j++){
                         System.out.print("ox");
                     }
                 }
                 System.out.println();
             }
-        } catch (Exception e){
+        } catch (NumberFormatException IllegalArgumentException) {
```

Comment:
why is there different exception name and type? Also i think there will be InputMissmatchException.